### PR TITLE
CNV-18515: more placeholders

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.13

Issue: [CNV-18515](https://issues.redhat.com//browse/CNV-18515)

Link to docs preview: N/A
(Addition of release note placeholders only; no visible changes in preview content)

QE review: N/A
(Addition of release note placeholders only; no visible changes in preview content)

### CODE REVIEW

**Comment by aspauldi** [comment]
Cancel /label peer-review-needed for now.

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -78,6 +78,7 @@ The SVVP Certification applies to:
 === Storage
 
 //CNV-18462 Release note: CHANGE Retire alpha APIs for Storage
+//CNV-18462 Release note: CHANGE  - Retire Storage Alpha APIs
```
Comment (kalexand-rh) [inline virt/virt-4-13-release-notes.adoc:81]: These first two are duplicates. It otherwise LGTM.

File: virt/virt-4-13-release-notes.adoc [hunk 2]
```diff
@@ -118,6 +121,8 @@ Removed features are not supported in the current release.
 
 //CNV-18462 Release note: CHANGE  Retire alpha APIs for Storage
 
+//CNV-19043 Release note: CHANGE Remove rhel6 support
```
Comment (ousleyp) [inline virt/virt-4-13-release-notes.adoc:124]: The RN for this placeholder is on line 118; you might want to move it up or remove it, depending on how you want to handle the placeholders. :)

File: virt/virt-4-13-release-notes.adoc [hunk 3]
```diff
@@ -90,6 +91,8 @@ The SVVP Certification applies to:
 
 //CNV-25574 Release note: NEW Ability to make a hot-plug disk part of the VM
 
+//CNV-22574 Release note: NEW UI - Ability to make a hot-plug disk part of the VM
```
Comment (ousleyp) [inline virt/virt-4-13-release-notes.adoc:94]: Looks like a duplicate of the placeholder on line 92?

File: virt/virt-4-13-release-notes.adoc (hunk 1 above)
Comment (ousleyp) [inline virt/virt-4-13-release-notes.adoc:81]: There is also an existing placeholder for this on line 122.

File: virt/virt-4-13-release-notes.adoc [hunk 4]
```diff
@@ -168,13 +173,8 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 //CNV-20965 Release note: PREVIEW Default creation and deployment of common set of instancetypes and preferences that eventually replace common templates
 
-//CNV-21922 Release note: PREVIEW (Looks like a Dev preview so maybe no doc) DevPrev: Instancetype and Preference inference based on Volume/PVC annotations
-
 //CNV-21991 Release note: PREVIEW CNV Provider for Hypershift
 
-//CNV-21994 Release note: PREVIEW Create a `VirtualMachine` definition from the CLI supplying only an instancetype with an optional set of preferences, volumes, and networks
```
Comment (ousleyp) [inline virt/virt-4-13-release-notes.adoc:175]: Did you mean to remove these placeholders (for CNV-21922 and CNV-21994)?

**Review by ousleyp — COMMENTED** [review]
Just noting a few things that may have gone awry when resolving merge conflicts. :)

File: virt/virt-4-13-release-notes.adoc (hunk 4 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:175]: I did mean to remove them. CNV-21992 and CNV-21994 were closed I think by Avital as Won't Do.

File: virt/virt-4-13-release-notes.adoc (hunk 2 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:124]: I moved the placeholder up.

File: virt/virt-4-13-release-notes.adoc (hunk 1 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:81]: I removed one of the instances in the file.

File: virt/virt-4-13-release-notes.adoc (hunk 3 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:94]: I removed one of the instances from the file.

Comment (ousleyp) [reply virt/virt-4-13-release-notes.adoc:92]: It looks like CNV-2**5**574 is the correct Jira ID for this one, not CNV-2**2**574.

**Review by ousleyp — COMMENTED** [review]
Just one nit with a Jira ID. Otherwise, this LGTM

File: virt/virt-4-13-release-notes.adoc (hunk 3 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:92]: Fixed it; thanks!

**Pull request merged**